src/main/java/com/example/bookstoreweb/service/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,112 @@
+package com.example.bookstoreweb.service.impl;
+
+import com.example.bookstoreweb.dto.order.OrderItemDto;
+import com.example.bookstoreweb.dto.order.OrderRequestDto;
+import com.example.bookstoreweb.dto.order.OrderRequestUpdateDto;
+import com.example.bookstoreweb.dto.order.OrderResponseDto;
+import com.example.bookstoreweb.dto.user.UserResponseDto;
+import com.example.bookstoreweb.exception.EntityNotFoundException;
+import com.example.bookstoreweb.mapper.OrderMapper;
+import com.example.bookstoreweb.model.CartItem;
+import com.example.bookstoreweb.model.Order;
+import com.example.bookstoreweb.model.OrderItem;
+import com.example.bookstoreweb.model.ShoppingCart;
+import com.example.bookstoreweb.repository.OrderItemRepository;
+import com.example.bookstoreweb.repository.OrderRepository;
+import com.example.bookstoreweb.repository.ShoppingCartRepository;
+import com.example.bookstoreweb.service.OrderService;
+import com.example.bookstoreweb.service.UserService;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Objects;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+public class OrderServiceImpl implements OrderService {
+    private final OrderRepository orderRepository;
+    private final OrderItemRepository orderItemRepository;
+    private final OrderMapper orderMapper;
+    private final UserService userService;
+    private final ShoppingCartRepository shoppingCartRepository;
+
+    @Override
+    public List<OrderResponseDto> findAllOrdersByUser(String email, Pageable pageable) {
+        UserResponseDto responseDto = userService.getByEmail(email);
+        return orderRepository.findAllByUserId(responseDto.getId())
+                .stream()
```

Comment:
```suggestion
        return orderRepository.findAllByUserId(responseDto.getId()).stream()
```
make sure all streams in you application is like that.